src/navigation/drive.py

```diff
@@ -138,6 +171,7 @@ def get_drive_command(
         :param turn_in_place_thresh: The angle threshold to consider the rover facing the target position and ready to drive forward towards it.
         :param in_odom: Whether to use odom constants or map constants.
         :param drive_back: True if rover should drive backwards, false otherwise.
+        :param prev_target: The previous target position, used to compute the lookahead point. (this is optional, if not provided then the lookahead point will be the target position)
```

Comment:
what's the difference between prev_target and self._last_target?

Reply:
self._last_target will always be the last target passed into the function so it may be like the target itself slightly moving, however prev_target is passed by the caller and is really meant to signify the last location from which the rover is coming to form a path that the rover is trying to follow. The distinction is important because if you just always use self.last_target to do the computation you will think the path you are trying to follow is in between two AR tag locations for instance. I agree that this probably isn't obvious, I could add a comment explaining like above or refactor to try to make more clear if you want.

Reply:
ok I think u should definitely rename these variables because last and previous pretty much mean the same thing so it's confusing

Reply:
ok, renamed